operator/src/simulation/mod.rs

```diff
@@ -0,0 +1,58 @@
+//! Simulation is a k8s custom resource that defines a Ceramic simulation.
+pub(crate) mod worker;
+pub(crate) mod manager;
+pub(crate) mod controller;
+
+pub use controller::run;
+
+use kube::CustomResource;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+/// Primary CRD for creating and managing a Ceramic Simulation.
+#[derive(CustomResource, Serialize, Deserialize, Debug, PartialEq, Clone, JsonSchema)]
+#[kube(
+    group = "keramik.3box.io",
+    version = "v1",
+    kind = "Simulation",
+    plural = "simulations",
+    status = "SimulationStatus",
+    derive = "PartialEq"
```

Comment:
I think we should make the simulation resource namespaced, that way we don't have to find the namespace from the network.